IdentityCore/src/MSIDError.h

```diff
@@ -248,6 +248,30 @@ typedef NS_ENUM(NSInteger, MSIDErrorCode)
     // SSO Extension internal error
     MSIDErrorSSOExtensionUnexpectedError           =   -51815,
     
+    // JIT - Link - Timeout while waiting for server confirmation
+    MSIDErrorJITLinkServerConfirmationTimeout      =   -51816,
```

Comment:
You need to map those in MSAL error converter as well

Reply:
I have updated the MSAL repository with the mapping: https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/1529